src/app/api/payments/route.ts

```
@@ -0,0 +1,165 @@
+    }
+
+    console.info("Arcjet checks passed", decision.id);
+
+    // This is where you would generate the checkout link. Return it to the form
+    // component to redirect the user to the payment page. See
+    // https://docs.stripe.com/checkout/quickstart?lang=node&client=next for an
+    // example with Stripe Checkout
+
+    return NextResponse.json({
+      success: true,
+      paymentLink: "https://www.example.com",
```

ESLint

[error] Strings must use singlequote.

(style/quotes)